### PR TITLE
Fix golint failures: pkg/ssh

### PR DESCRIPTION
**Pull request by mdeggies**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This resolves the golint errors under `pkg/ssh`

**Which issue(s) this PR fixes**:

Fixes # https://github.com/kubernetes/kubernetes/issues/68026

**Does this PR introduce a user-facing change?**:

NONE

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by mdeggies** [comment]
Signed the CLA

File: pkg/ssh/ssh.go [hunk 1]
```diff
@@ -112,15 +112,15 @@ func (s *sshTunnel) Open() error {
...
+		return nil, errors.New("tunnel is not opened")
 	}
 	// This Dial method does not allow to pass a context unfortunately
 	return s.client.Dial(network, address)
 }
 
 func (s *sshTunnel) Close() error {
 	if s.client == nil {
-		return errors.New("Cannot close tunnel. Tunnel was not opened.")
+		return errors.New("cannot close tunnel. tunnel was not opened")
```
Comment (spiffxp) [inline pkg/ssh/ssh.go:123]: parity with Dial's err message
```suggestion
		return errors.New("cannot close tunnel: tunnel is not opened")
```

**Review by spiffxp — COMMENTED** [review]
one nit

**Comment by mdeggies** [comment]
Thanks for your review :-D

**Comment by mdeggies** [comment]
@Jefftree I think there was a flakey test that failed the first two times this ran. It seems to be passing now- let me know if it looks G2G or if I need to make any changes! Thanks

File: pkg/ssh/ssh.go [hunk 2]
```diff
@@ -376,7 +386,9 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 	go l.createAndAddTunnel(e.Address)
 }
 
-func (l *SSHTunnelList) Dial(ctx context.Context, net, addr string) (net.Conn, error) {
+// Dial returns a new Connection interface for the input tunnel. If the tunnel is not
+// open, an error is returned.
```
Comment (Jefftree) [inline pkg/ssh/ssh.go:390]: nit: The function also connects to the corresponding addr through a tunnel in the TunnelList.

**Review by Jefftree — COMMENTED** [review]
Thanks for the PR! One nit otherwise lgtm :)

File: pkg/ssh/ssh.go (hunk 2 above)
Comment (mdeggies) [reply pkg/ssh/ssh.go]: Thanks Jeff! Fixed in the latest commit.

File: test/utils/conditions.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ package utils
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (spiffxp) [inline test/utils/conditions.go:22]: I would remove this file from the PR, you're fixing a pkg/ lint, I'm not sure why a test/ change is sneaking in

**Comment by lavalamp** [comment]
sorry I didn't see this until now -- please ping me next time :)

**Pull request closed without merging**